src/main/java/racingcar/CarRace.java

```diff
@@ -46,7 +46,32 @@ public int getTryCnt() {
         return raceResult;
     }
 
+    public List<Car> getWinnerCars() {
+        List<Car> winnerCar = new ArrayList<>();
+        int winnerDistance = getWinnerDistance(raceResult);
+        RaceSnapshot lastRaceSnapShot = raceResult.get(raceResult.size()-1);
+        for(Car car : lastRaceSnapShot.cars){
+            if(car.getDistance() == winnerDistance){
+                winnerCar.add(car);
+            }
+        }
+        return winnerCar;
+    }
+
+    public int getWinnerDistance(List<RaceSnapshot> raceSnapshotList) {
+        RaceSnapshot lastRaceSnapShot = raceSnapshotList.get(raceSnapshotList.size()-1);
+        final Comparator<Integer> comp1 = (s1, s2) -> s1.compareTo(s2);
+        int winnerDistance = lastRaceSnapShot.cars.stream()
+                .max((x, y) -> x.getDistance() - y.getDistance())
+                .get().getDistance();
+        return winnerDistance;
+    }
```

Comment:
```suggestion
    public int getWinnerDistance(RaceSnapshot lastRaceSnapShot) {
        return lastRaceSnapShot.cars.stream()
                .max(Comparator.comparingInt(Car::getDistance))
                .get().getDistance();
    }
```
가독성을 좀 더 높여줄 수 있을거 같아요! 😄